test/units/module_utils/basic/test_set_mode_if_different.py

```diff
@@ -89,6 +92,7 @@ def test_mode_unchanged_when_already_0660(am, mock_stats, mocker, mode, check_mo
     am.check_mode = check_mode
     mocker.patch('os.lstat', side_effect=[mock_stats['after'], mock_stats['after'], mock_stats['after']])
     m_lchmod = mocker.patch('os.lchmod', return_value=None, create=True)
+    mocker.patch('os.path.exists', return_value=True)
```

Comment:
For consistency, shouldn't you assert that this is called, like os.lchmod ?

Reply:
Following @Spredzy  advice, in order to be consistent with mock of `os.lstat`, `assert not m_path_exists.called` was removed :)
We kept it only when `return_value` of the `os.path.exists` mock is parametrized.

Adding this assert for every `os.path.exists` mock doesn't hurt, what do you think ?